keen/api.py

```diff
@@ -190,6 +211,45 @@ def delete_events(self, event_collection, params):
         self.error_handling(response)
         return True
 
+    def get_collection(self, collection_name):
+        """
+        Extracts info about a collection using the Keen IO API. A master key must be set first.
+
+        :param event_name: the name of the collection to retrieve info for
```

Comment:
Could you change "event_name" to "event_collection" so it's consistent with the rest of the doc strings?